html_tag_collector/collector.py

```diff
@@ -96,7 +91,8 @@ async def run_get_response(urls):
     """
     tasks = []
     urllib3.disable_warnings()
-    session = AsyncHTMLSession(workers=100)
+    user_agent = "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36"
```

Comment:
Like I said above, including this line of code will, combined with the similar string setting in the RootURLCache, mean that should this PR be accepted as-is, the same quite long and precise string will be hardcoded three times throughout the repo. I think it makes sense to extract all three instances to a small function called "get_user_agent()"